doc/modules/pipeline.rst

```diff
@@ -269,3 +269,59 @@ and ignored by setting to ``None``::
 
  * :ref:`sphx_glr_auto_examples_plot_feature_stacker.py`
  * :ref:`sphx_glr_auto_examples_hetero_feature_union.py`
+
+.. _frozen:
+
+Frozen estimators
+=================
+.. currentmodule:: sklearn
+
+It can be useful to pre-fit an estimator before including it in a Pipeline,
+FeatureUnion or other meta-estimators.  Example applications include:
+
+* transfer learning: incorporating a transformer trained on a large unlabelled
+  dataset in a prediction pipeline where the data to be modelled is much smaller
+* feature selection on the basis of an already fitted predictive model
+* calibrating an already fitted classifier for probabilistic output
+
+To enable this, some meta-estimators will refrain from fitting an estimator
+where it has the attribute ``frozen`` set to ``True``. For example::
+
+    Without transfer learning
+
+    >>> from sklearn.datasets import load_...
+    >>> from sklearn.model_selection import cross_val_score
+    >>> cross_val_score(make_pipeline(TfidfVectorizer(), LogisticRegression()),
+    ...                 X, y)
+
+    With transfer learning:
+    >>> tfidf = TfidfVectorizer().fit(large_X)
+    >>> tfidf.frozen = True
```

Comment:
I don't think a method is helpful either. Only makes it harder to use something that for whatever strange reason does not inherit from base